ruff_lsp/server.py

```diff
@@ -558,6 +558,14 @@ async def _lint_document_impl(document: Document) -> list[Diagnostic]:
     result = await _run_check_on_document(document)
     if result is None:
         return []
+
+    # For `ruff check`, 0 indicates successful completion with no remaining
+    # diagnostics, 1 indicates successful completion with remaining diagnostics,
+    # and 2 indicates an error.
+    if result.exit_code != 0 and result.exit_code != 1:
```

Comment:
Nit: I don't know what's more idiomatic Python
```suggestion
    if not result.exit_code in [0, 1]:
```

Reply:
😬 
```python
if result.exit_code not in [0, 1]:
	...
```

Reply:
I think `not in (0, 1)` is most idiomatic, probably.

Reply:
> 😬
> 
> ```python
> if result.exit_code not in [0, 1]:
> 	...
> ```

Python's hard

Reply:
Haha both work!